internal/tools/pipeline/providers/cron/cron.service.go

```diff
@@ -46,6 +47,27 @@ func (s *provider) CronCreate(ctx context.Context, req *pb.CronCreateRequest) (*
 		return nil, apierrors.ErrCreatePipelineCron.InvalidParameter(errors.Errorf("missing pipelineYml"))
 	}
 
+	// Get User Info if not exist
+	if _, ok := req.NormalLabels[apistructs.LabelUserID]; !ok {
+		if req.IdentityInfo != nil && req.IdentityInfo.UserID != "" {
+			req.NormalLabels[apistructs.LabelUserID] = req.IdentityInfo.UserID
+		}
+	}
+
+	// Get Owner Info
+	if _, ok := req.FilterLabels[apistructs.LabelOwnerUserID]; !ok {
+		if req.OwnerUser != nil && req.OwnerUser.ID != nil {
+			ownerId := req.OwnerUser.ID.GetStringValue()
+			if ownerId == "" {
+				ownerId = conf.InternalUserID()
+			}
+			req.FilterLabels[apistructs.LabelOwnerUserID] = ownerId
+		}
+	}
+	if req.NormalLabels[apistructs.LabelOwnerUserID] != req.FilterLabels[apistructs.LabelOwnerUserID] {
```

Comment:
Confirm the purpose of owner id.

Reply:
The owner parameter is used to create a pipeline_extra record when starting a scheduled pipeline